amr-wind/incflo.H

```diff
@@ -172,6 +174,9 @@ private:
     // Prescribe advection velocity
     bool m_prescribe_vel = false;
 
+    // Advection iterations every timestep
```

Comment:
maybe something with non_linear_iterations in the name? I am a bit wary of our naming for these things because we often mean different things when we say "nonlinear". I am not sure what the right name is but we need to be clear and make sure we stick with it. I have no problem with long, explicit names: e.g. `m_non_linear_advection_iterations`.  Let's add some documentation too.

Reply:
We could go for something like `m_outer_loop_iterations`. I am going to modify the loops to make this compatible with exawind-driver, and I think that is going to make it clear that these are outer loop iterations